tests/end2end/features/hints.feature

```diff
@@ -231,6 +241,7 @@ Feature: Using hints
         And I run :tab-only
         And I hint with args "links tab" and follow a
         And I wait until data/hello.txt is loaded
+        And I wait 0.5s
```

Comment:
For some reason, this seems to be race-conditioning, in the previous few commits, it wasn't waiting long enough to get the tab into the saved session. Adding this fixes it, but it's not a good solution so it should be properly looked at...

Reply:
Worked around in 0bdee1e2920b3a905205b5cf5f4ee4f0cc83f46f - see the commit message for details.